h/services/links.py

```diff
@@ -63,9 +63,9 @@ def get_all(self, annotation):
         for name, (g, hidden) in self.registry[LINK_GENERATORS_KEY].items():
             if hidden:
                 continue
-            l = g(self._request, annotation)
```

Comment:
Total aside here: this one was interesting to me. At first I thought it was enforcing no-single-lett-er-variable-names and I was rejoicing to the heavens because that is a _bete noir_ for me, but then I realized that the error raised is `E741 — ambiguous variable name 'l'`; further Googling leads me to the "real" rule here, which is:

> do not use variables named ‘l’, ‘O’, or ‘I’

I just found this interesting—the error message isn't necessarily the same in all contexts